src/crypto/fs/blake3.rs

```diff
@@ -0,0 +1,46 @@
+use ark_crypto_primitives::sponge::{Absorb, CryptographicSponge};
```

Comment:
@wizardofmenlo keep which hashers?

Reply:
Do we need any hasher if we move to nimue?

Reply:
^ https://github.com/WizardOfMenlo/stir/blob/main/src/crypto/merkle_tree/blake3.rs